### ISSUE TITLE
Should not lint on `dplyr` columns

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#15]
## box.linters version

0.0.0.9006

## Source code to lint

```R
box::use(
  dplyr[`%>%`, filter, select],
)

mtcars %>%
  select(mpg, cyl) %>%
  filter(mpg <= 10)
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Expected result

Should not lint on columns (`mpg`, `cyl`).
